src/Symfony/Component/Form/FormFactory.php

```diff
@@ -61,36 +61,12 @@ public function createForProperty($class, $property, $data = null, array $option
      */
     public function createBuilder($type = 'Symfony\Component\Form\Extension\Core\Type\FormType', $data = null, array $options = array())
     {
-        $name = null;
-        $typeName = null;
-
         if ($type instanceof ResolvedFormTypeInterface) {
```

Comment:
I'm not sure we actually need to keep support for `ResolvedFormTypeInterface` in the factory. it is not supported according to FormFactoryInterface